tests/unit/service/test_feature_materialize.py

```diff
@@ -314,7 +314,7 @@ async def test_scheduled_materialize_features(
     """
     await feature_materialize_service.scheduled_materialize_features(offline_store_feature_table)
 
-    executed_queries = extract_session_executed_queries(mock_snowflake_session, "execute_query")
```

Comment:
@gee-senbong These cause the fixture files to change. Previously the fixture files only capture queries executed using `execute_query()`. Now the fixture files capture `execute_query_long_running()`, which covers more queries than before.